packages/communication-react/breaking-change-check/checkBreakingChanges.ts

```diff
@@ -78,7 +78,7 @@ type BreakingChangeCallClientStateTest = CheckBreakingChanges<
 >;
 
 // Stateful client test start
-
+// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-constraint
 type StatefulClientTestFunction<Client extends any> = (client: NestedPartial<OmitState<Partial<Client>>>) => Client;
```

Comment:
Can we just remove `extends any` here?

Reply:
this would be a question for @PorterNan. IIRC, the casting was part of the determining if there is a break.